tests/test_single_topology.py

```diff
@@ -1708,14 +1708,15 @@ def test_hif2a_end_state_symmetry_nightly_test():
             amm_rev = AtomMapMixin(mol_b, mol_a, core[:, ::-1])
             fused_map = np.concatenate(
                 [
-                    np.array([[x, y] for x, y in zip(amm_fwd.a_to_c, amm_rev.b_to_c)]),
+                    np.array([[x, y] for x, y in zip(amm_fwd.a_to_c, amm_rev.b_to_c)], dtype=np.int32).reshape(-1, 2),
                     np.array(
                         [
                             [x, y]
                             for x, y in zip(amm_fwd.b_to_c, amm_rev.a_to_c)
                             if x not in core[:, 0] and y not in core[:, 1]
-                        ]
-                    ),
+                        ],
+                        dtype=np.int32,
+                    ).reshape(-1, 2),
```

Comment:
Note that a similar `fused_map_from(amm_fwd, amm_rev)` snippet appears in another unit test: 

https://github.com/proteneer/timemachine/blob/70bec1d7db4d0ccc0ab7651f81b5d3990e4446d4/tests/test_single_topology.py#L1741-L1748

This zero-length corner case isn't encountered in that unit test, but it might still be good to extract a single copy of this utility later.